src/main/java/gov/usgs/wma/statistics/control/StatsService.java

```diff
@@ -187,7 +186,7 @@ protected void validateParamMedians(String medians, JsonDataBuilder builder) {
 				String val  = cols[1].trim();
 				
 				BigDecimal value = new BigDecimal(val);
-				WLSample sample = new WLSample(time, value, "ft", value, "", true, "", value);
```

Comment:
true was a placeholder in early development that should have been replaced with mediation type.

Reply:
Can we call it something like `sortedMediationIsEqual` (maybe not exactly that, but something to indicate a true|false condition).

Reply:
I called it equalSortOrder.